app/views/layouts/application.html.erb

```diff
@@ -3,7 +3,11 @@
 <head>
   <title>Chess</title>
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
```

Comment:
you can leave the first part, only need to remove the turbolinks part. same goes for the stylesheet link in line 5